### PR TITLE
[FIX] account_edi_ubl_cii,l10n_it_edi: fix import and embed pdf to efff

### PR DESCRIPTION
**Pull request by JulienVR**
Fix E-FFF: it is no longer handled by l10n_be_edi, the addition of
the pdf inside the xml should occur in the account_edi_ubl_cii module.
See https://github.com/odoo/odoo/commit/4cdd9a0d86d466f200dddd52bca8f48449834f82

In addition, fix the import of EDI documents is broken since it's no
longer possible to write on invisible fields.
Add checks before assigning product_uom_id or currency_id.
See https://github.com/odoo/odoo/commit/5ccc32fcf72cbfb6bb077d99a7657416c502aac1

### CODE REVIEW

File: addons/l10n_it_edi/models/account_edi_format.py [hunk 1]
```diff
@@ -401,7 +401,10 @@ def _import_fattura_pa(self, tree, invoice):
...
             if not company:
                 elements = tree.xpath('//CessionarioCommittente//CodiceFiscale')
-                company = elements and self.env['res.company'].search([('l10n_it_codice_fiscale', 'ilike', elements[0].text)], limit=1)
+                if not elements:
+                    _logger.warning('No Codice Fiscale found in the xml.')
+                    continue
+                company = self.env['res.company'].search([('l10n_it_codice_fiscale', 'ilike', elements[0].text)], limit=1)
                 if not company:
                     # Only invoices with a correct VAT or Codice Fiscale can be imported
                     _logger.warning('No company found with VAT or Codice Fiscale like %r.', elements[0].text)
```
Comment (william-andre) [inline addons/l10n_it_edi/models/account_edi_format.py:410]: Wait, so this is just failing silently? (from the user PoV)
Doesn't `account_edi` have a warning/error management mechanism?

**Review by william-andre — COMMENTED** [review]
Please split the commits
* one for the invisible fields thingy
* one for efff
* one for adding the warning on the codice [1]

[1]: I think that one should be removed and made correctly in another PR with proper warnings or whatever so that the user notices that something is missing

_______

In you commit message, you are referencing pull requests, which will spam those at each commit done in the staging builds and at every forward port.
Please reference the commit (in `odoo`, not `odoo-dev`) instead

File: addons/l10n_it_edi/models/account_edi_format.py (hunk 1 above)
Comment (JulienVR) [reply addons/l10n_it_edi/models/account_edi_format.py]: The warning/error management you're refering to is only available when generating an edi attachment, not for import, if I'm not missing sth :slightly_smiling_face:. I'll open a PR and log the messages in the chatter (I discussed it with Josse).

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by william-andre** [comment]
@robodoo rebase-ff

**Pull request closed without merging**